selenium/src/main/java/com/deque/html/axecore/selenium/AxeBuilder.java

```diff
@@ -739,10 +744,25 @@ private Results analyzePost43x(final WebDriver webDriver, final Object rawContex
 
     String prevWindow = WebDriverExtensions.openAboutBlank(webDriver);
     injectAxe(webDriver);
+    String partialResString = "";

```

Comment:
nit - 

This function is already quite large, breaking it out into its own function would allow commenting on what and why it was needed and can reference [proposal](https://github.com/dequelabs/axe-api-team/pull/312)